README.md

```diff
@@ -269,106 +269,74 @@ The built-in multi-cursor support may not meet your needs. Please refer to the p
         -   `[start_line, start_character, end_line, end_character]`
         -   `{start = { line = start_line, character = start_character}, end = { line = end_line, character = end_character}}`
     -   `restore_selection` (boolean): Whether to preserve the current selection. Only valid when `range` is specified.
         Defaults to `true`.
-    -   `callback`: Optional callback function to handle the action result. The callback function should have the
-        following signature: `function(err: string|nil, ret: any)`. The first argument is the error message, and the
-        second is the result. If no callback is provided, any error message will be shown as a notification in VSCode.
+    -   `callback`: Function to handle the action result. Must have this signature:
+        ```lua
+        function(err: string|nil, ret: any)
+        ```
+        -   `err` is the error message, if any
+        -   `ret` is the result
+        -   If no callback is provided, error will be shown as a VSCode notification.
 
-##### `vscode.call(name, opts, timeout)`
+### vscode.call(name, opts, timeout)
 
-This function is used to run an action synchronously.
+Synchronously executes a vscode command.
 
 Parameters:
 
 -   `name` (string): The name of the action, generally a vscode command.
--   `opts` (table): Optional table of options. All fields in the table are optional.
-    -   `args` (table): Optional arguments for the action.
-    -   `range` (table): Specific range for the action. In visual mode, this parameter is generally not needed. There
-        are three supported formats for defining the range (all values are 0-indexed):
-        -   `[start_line, end_line]`
-        -   `[start_line, start_character, end_line, end_character]`
-        -   `{start = { line = start_line, character = start_character}, end = { line = end_line, character = end_character}}`
-    -   `restore_selection` (boolean): Whether to preserve the current selection. Only valid when `range` is specified.
-        Defaults to `true`.
+-   `opts` (table): Same as [vscode.action()](#vscodeactionname-opts).
 -   `timeout` (number): Timeout in milliseconds. The default value is -1, which means there is no timeout.
 
 Returns: the result of the action
 
-##### Examples
+#### Examples
+
+- Format selection (default binding):
+  ```vim
+  xnoremap = <Cmd>lua require('vscode-neovim').call('editor.action.formatSelection')<CR>
+  nnoremap = <Cmd>lua require('vscode-neovim').call('editor.action.formatSelection')<CR><Esc>
+  nnoremap == <Cmd>lua require('vscode-neovim').call('editor.action.formatSelection')<CR>
+  ```
+- Open definition aside (default binding):
+  ```vim
+  nnoremap <C-w>gd <Cmd>lua require('vscode-neovim').action('editor.action.revealDefinitionAside')<CR>
+  ```
+- Find in files for word under cursor:
+  ```vim
+  nnoremap ? <Cmd>lua require('vscode-neovim').action('workbench.action.findInFiles', { args = { { query = vim.fn.expand('<cword>') } } })<CR>
```

Comment:
@xiyaowong why is `args` a list instead of a map? vscode commands accept URL-like querystrings, so order shouldn't matter should it?

Reply:
![image](https://github.com/vscode-neovim/vscode-neovim/assets/47070852/dd851dde-9d54-4f86-bc0a-56773c98a115)

Reply:
I don't understand what you mean, can you give an example? This corresponds to the command function parameters.